Spinner/Spinner.js

```diff
@@ -57,17 +63,29 @@ const SpinnerCore = kind({
 	},
 
 	computed: {
+		'aria-label': ({'aria-label': label, type}) => {
+			// TODO: These static values will need to be localized
+			return label || type === 'searching' ? 'Searching' : 'Loading';
```

Comment:
We'll eventually need to support localization so I've refactored this into a computed prop rather than assigning the `type` directly as `aria-label`.